src/main/scala/com/advancedtelematic/director/db/Repository.scala

```diff
@@ -97,16 +102,27 @@ protected class AdminRepository()(implicit db: Database, ec: ExecutionContext) e
       .paginateResult(offset = offset, limit = limit)
   }
 
-  def findDevice(namespace: Namespace, device: DeviceId): Future[Seq[EcuInfoResponse]] = db.run {
+  def findDeviceById(device: DeviceId): Future[Seq[EcuInfoResponse]] = db.run {
```

Comment:
This is finding `EcuInfoResponse`s, not a device, right?

Reply:
yeah but this nomenclature was already there for whatever reason and I didn't want to disturb the monster.

Reply:
haha fair enough 🙈